modules/nw-networkpolicy-about.adoc

```diff
@@ -123,3 +123,53 @@ spec:
 `NetworkPolicy` objects are additive, which means you can combine multiple `NetworkPolicy` objects together to satisfy complex network requirements.
 
 For example, for the `NetworkPolicy` objects defined in previous samples, you can define both `allow-same-namespace` and `allow-http-and-https` policies within the same project. Thus allowing the pods with the label `role=frontend`, to accept any connection allowed by each policy. That is, connections on any port from pods in the same namespace, and connections on ports `80` and `443` from pods in any namespace.
+
+[id="nw-networkpolicy-allow-from-router_{context}"]
+== allow from router
+
+Allows traffic from router regardless of the router configuration:
+
+To make a project allow external traffic regardless of the configuration, add the following `NetworkPolicy` object.
+
+`policy-group.network.openshift.io/ingress` label supports both Openshift-SDN and OVN-Kubernetes. 
```

Comment:
I think this probably needs to be a <1> and under the yaml block.

Reply:
Done.